index.php

```diff
@@ -0,0 +1,311 @@
+<?php
+    if(!$_GET["page"]){
+        header("location:index.php?page=home");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function header() is discouraged](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)